### DISCUSSION TITLE
Cannot resolve keyword 'port_list' into field.

### DISCUSSION THREAD

**Discussion by Jkasnw-7823** [netbox-community/netbox#9308] [Q&A]
I have installed netbox 3.2.0. After upgrading to 3.2.1, I got an exception for 'https://.../ipam/service-templates/'. Now after upgrade to 3.2.2 the exception raise farther.

### The exception from now

```console
<class 'django.core.exceptions.FieldError'>

Cannot resolve keyword 'port_list' into field. Choices are: created, custom_field_data, description, id, journal_entries, last_updated, name, ports, protocol, tagged_items, tags

Python version: 3.8.10
NetBox version: 3.2.2
```

In my opinion it seems to be a database configuration error, lost column or something else. Could anybody help?

**Comment by wtgitz** [comment]
I'm getting the same error.
Python version: 3.8.10
NetBox version: 3.2.4

**Comment by candlerb** [comment]
Please provide a full backtrace.  You can get this by setting the EMAIL and ADMINS settings in configuration.py; then when the exception occurs, it will be E-mailed to you.

**Reply by wtgitz** [reply]
Here's the full traceback.

`Environment:


Request Method: GET
Request URL: https://netbox.lcccorp.com/ipam/service-templates/

Django Version: 4.0.4
Python Version: 3.8.10
Installed Applications:
['django.contrib.admin',
 'django.contrib.auth',
 'django.contrib.contenttypes',
 'django.contrib.sessions',
 'django.contrib.messages',
 'django.contrib.staticfiles',
 'django.contrib.humanize',
 'corsheaders',
 'debug_toolbar',
 'graphiql_debug_toolbar',
 'django_filters',
 'django_tables2',
 'django_prometheus',
 'graphene_django',
 'mptt',
 'rest_framework',
 'social_django',
 'taggit',
 'timezone_field',
 'circuits',
 'dcim',
 'ipam',
 'extras',
 'tenancy',
 'users',
 'utilities',
 'virtualization',
 'wireless',
 'django_rq',
 'drf_yasg']
Installed Middleware:
['graphiql_debug_toolbar.middleware.DebugToolbarMiddleware',
 'django_prometheus.middleware.PrometheusBeforeMiddleware',
 'corsheaders.middleware.CorsMiddleware',
 'django.contrib.sessions.middleware.SessionMiddleware',
 'django.middleware.common.CommonMiddleware',
 'django.middleware.csrf.CsrfViewMiddleware',
 'django.contrib.auth.middleware.AuthenticationMiddleware',
 'django.contrib.messages.middleware.MessageMiddleware',
 'django.middleware.clickjacking.XFrameOptionsMiddleware',
 'django.middleware.security.SecurityMiddleware',
 'netbox.middleware.ExceptionHandlingMiddleware',
 'netbox.middleware.RemoteUserMiddleware',
 'netbox.middleware.LoginRequiredMiddleware',
 'netbox.middleware.DynamicConfigMiddleware',
 'netbox.middleware.APIVersionMiddleware',
 'netbox.middleware.ObjectChangeMiddleware',
 'django_prometheus.middleware.PrometheusAfterMiddleware']



Traceback (most recent call last):
  File "/opt/netbox/venv/lib/python3.8/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/opt/netbox/venv/lib/python3.8/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/opt/netbox/venv/lib/python3.8/site-packages/django/views/generic/base.py", line 84, in view
    return self.dispatch(request, *args, **kwargs)
  File "/opt/netbox/netbox/utilities/views.py", line 90, in dispatch
    return super().dispatch(request, *args, **kwargs)
  File "/opt/netbox/venv/lib/python3.8/site-packages/django/views/generic/base.py", line 119, in dispatch
    return handler(request, *args, **kwargs)
  File "/opt/netbox/netbox/netbox/views/generic/bulk_views.py", line 185, in get
    table.configure(request)
  File "/opt/netbox/netbox/netbox/tables/tables.py", line 142, in configure
    self.order_by = ordering
  File "/opt/netbox/venv/lib/python3.8/site-packages/django_tables2/tables.py", line 543, in order_by
    self.data.order_by(self._order_by)
  File "/opt/netbox/venv/lib/python3.8/site-packages/django_tables2/data.py", line 226, in order_by
    self.data = self.data.order_by(*order_by_accessors)
  File "/opt/netbox/venv/lib/python3.8/site-packages/django/db/models/query.py", line 1295, in order_by
    obj.query.add_ordering(*field_names)
  File "/opt/netbox/venv/lib/python3.8/site-packages/django/db/models/sql/query.py", line 2167, in add_ordering
    self.names_to_path(item.split(LOOKUP_SEP), self.model._meta)
  File "/opt/netbox/venv/lib/python3.8/site-packages/django/db/models/sql/query.py", line 1677, in names_to_path
    raise FieldError(

Exception Type: FieldError at /ipam/service-templates/
Exception Value: Cannot resolve keyword 'port_list' into field. Choices are: created, custom_field_data, description, id, journal_entries, last_updated, name, ports, protocol, tagged_items, tags
`

**Reply by candlerb** [reply]
I think you have hit bug #9480, which was fixed in Netbox 3.2.5. It gives exactly the same error, and the diff [here](https://github.com/netbox-community/netbox/pull/9483/files) changes `port_list` to `ports`.

So please update to 3.2.5 and try again.

If you can't do that, then try the workaround given here: https://github.com/netbox-community/netbox/issues/9480#issuecomment-1146685750
